cmd/convert.go

```diff
@@ -16,6 +16,89 @@ import (
 	"github.com/yuzutech/kroki-go"
 )
 
+// getSupportedImageFormats returns the list of all the supported image formats
+// Please note that not all image formats are available on all diagram types
+func getSupportedImageFormats() []kroki.ImageFormat {
+	return []kroki.ImageFormat{
+		kroki.SVG,
+		kroki.PNG,
+		kroki.JPEG,
+		kroki.PDF,
+		kroki.Base64,
+	}
+}
+
+// getSupportedDiagramTypes returns the list of all supported diagram types
+func getSupportedDiagramTypes() []kroki.DiagramType {
+	return []kroki.DiagramType{
+		kroki.ActDiag,
+		kroki.BlockDiag,
+		kroki.BPMN,
+		kroki.Bytefield,
+		kroki.C4PlantUML,
+		kroki.Ditaa,
+		kroki.Erd,
+		kroki.Excalidraw,
+		kroki.GraphViz,
+		kroki.Mermaid,
+		kroki.Nomnoml,
+		kroki.NwDiag,
+		kroki.PacketDiag,
+		kroki.PlantUML,
+		kroki.RackDiag,
+		kroki.SeqDiag,
+		kroki.Svgbob,
+		kroki.UMlet,
+		kroki.Vega,
+		kroki.VegaLite,
+		kroki.WaveDrom,
+	}
+}
```

Comment:
Will be removed once https://github.com/yuzutech/kroki-go/pull/18 is merged and released